.coveragerc

```diff
@@ -1,2 +1,8 @@
 [run]
 branch = True
+
+[paths]
+source =
+    src/
```

Comment:
Why are we doing coverage of `src/`? I thought one of the points of this PR was to ensure that everything is executed out of `site-packages`.